File: app/src/main/java/com/neome/api/meta/base/dto/StudioMapOfTableHeader.java
// neome.ai API - do not change
//

package com.neome.api.meta.base.dto;

import java.util.Map;

import com.neome.api.meta.base.Types.MetaIdHeader;
import com.neome.api.meta.base.dto.StudioBase;
import com.neome.api.meta.base.dto.StudioDtoTableHeader;

@SuppressWarnings("unused")
public class StudioMapOfTableHeader extends StudioBase
{
  public MetaIdHeader[] keys;

  public Map<MetaIdHeader, StudioDtoTableHeader> map;
}

File: app/src/main/java/com/neome/api/meta/base/dto/StudioMapOfTableHeader.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.MetaIdHeader
import com.neome.api.meta.base.dto.StudioBase
import com.neome.api.meta.base.dto.StudioDtoTableHeader

interface StudioMapOfTableHeader : StudioBase
{
  val keys: List<MetaIdHeader>
  val map: Map<MetaIdHeader, StudioDtoTableHeader>
}